src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -468,6 +468,57 @@ bamm:EventShape
       ]
    ] .
 
+bamm:UnitShape
```

Comment:
Any reason why you did not add a shape for the Quantity Kind as well?

Reply:
Yes, because a Quantity Kind has only a `bamm:name` (which is handled by `NamedConceptShape`) and a `bamm:preferredName` (which is handled by `ConceptShape`) and nothing else.